baremetal_network_provisioning/db/bm_nw_provision_models.py

```diff
@@ -47,3 +47,56 @@ class HPIronicSwitchPortMapping(model_base.BASEV2):
     access_type = sa.Column(sa.String(36))
     segmentation_id = sa.Column(sa.Integer)
     host_id = sa.Column(sa.String(36))
+
+
+class BNPPhysicalSwitchPort(model_base.BASEV2, models_v2.HasId):
+    """Define physical switch port properties."""
+    __tablename__ = "bnp_physical_switch_ports"
+    switch_id = sa.Column(sa.String(255), nullable=False)
+    interface_name = sa.Column(sa.String(255), nullable=False)
+    ifindex = sa.Column(sa.String(255), nullable=False)
+    port_status = sa.Column(sa.String(16), nullable=False)
+    sa.PrimaryKeyConstraint('id')
+    sa.UniqueConstraint('switch_id', 'interface_name')
+
+
+class BNPPhysicalSwitch(model_base.BASEV2, models_v2.HasId):
+    """Define physical switch properties."""
+    __tablename__ = "bnp_physical_switches"
+    ip_address = sa.Column(sa.String(64), nullable=False)
+    mac_address = sa.Column(sa.String(32), nullable=True)
+    status = sa.Column(sa.String(16), nullable=False)
+    access_protocol = sa.Column(sa.String(16), nullable=False)
+    vendor = sa.Column(sa.String(16), nullable=False)
+    security_name = sa.Column(sa.String(255), nullable=True)
+    auth_protocol = sa.Column(sa.String(16), nullable=True)
+    auth_key = sa.Column(sa.String(255), nullable=True)
+    priv_protocol = sa.Column(sa.String(16), nullable=True)
+    priv_key = sa.Column(sa.String(255), nullable=True)
+    security_level = sa.Column(sa.String(16), nullable=True)
+    sa.PrimaryKeyConstraint('id')
+    sa.ForeignKeyConstraint('id', 'bnp_physical_switchports.switch_id',
+                            ondelete='CASCADE')
+
+
+class BNPSwitchPortMapping(model_base.BASEV2):
```

Comment:
Question : How will I add the LAG ports for the same neutron ID in this case? Since neutron_portid and switch_port_id kept unique .. If I want to map one neutron_port_id to multiple switch_port ? I am just thinking of accommodating for lag scenarios.